code/game/objects/structures/droppod.dm

```diff
@@ -215,30 +203,105 @@ GLOBAL_LIST_INIT(blocked_droppod_tiles, typecacheof(list(/turf/open/space/transi
 
 ///Do the stuff when it "hits the ground"
 /obj/structure/droppod/proc/dodrop(turf/targetturf, mob/user)
-	deadchat_broadcast(" has landed at [get_area(targetturf)]!", src, occupant)
+	deadchat_broadcast(" has landed at [get_area(targetturf)]!", src, user)
 	explosion(targetturf,-1,-1,2,-1)
 	playsound(targetturf, 'sound/effects/droppod_impact.ogg', 100)
 	QDEL_NULL(reserved_area)
 	addtimer(CALLBACK(src, .proc/completedrop, user), 7) //dramatic effect
 
-///completes everything after a dramatic effect
+///completes landing a little delayed for a dramatic effect
 /obj/structure/droppod/proc/completedrop(mob/user)
-	icon_state = "singlepod_open"
 	drop_state = DROPPOD_LANDED
-	exitpod()
+	update_icon()
+	for(var/atom/movable/deployed AS in contents)
+		deployed.forceMove(loc)
 
-/obj/structure/droppod/proc/exitpod(forced = FALSE)
-	SIGNAL_HANDLER
-	if(!occupant)
+
+/obj/structure/droppod/leader
+	name = "\improper TGMC Zeus command drop pod"
+	desc = "A menacing metal hunk of steel that is used by the TGMC for quick tactical redeployment. This one comes with command capabilities."
+	icon_state = "leaderpod"
+
+/obj/structure/droppod/leader/buckle_mob(mob/living/buckling_mob, force, check_loc, lying_buckle, hands_needed, target_hands_needed, silent)
+	. = ..()
+	if(!.)
+		return
+	if(buckling_mob.skills.getRating(SKILL_LEADERSHIP) < SKILL_LEAD_EXPERT)
+		balloon_alert(buckling_mob, "Can't use that!") // basically squad lead+ cant touch this
+		return FALSE
+
+/obj/structure/droppod/leader/set_target(new_x, new_y)
+	. = ..()
+	if(!.)
+		return
+	if(TIMER_COOLDOWN_CHECK(src, COOLDOWN_DROPPOD_TARGETTING))
+		addtimer(CALLBACK(src, /atom.proc/balloon_alert, buckled_mobs[1], "Target Assignment cooldown"), 7)
+		return
+	// this isnt the cheapest thing in the world so lets not let players spam it
+	TIMER_COOLDOWN_START(src, COOLDOWN_DROPPOD_TARGETTING, 10 SECONDS)
+
+	var/turf/target = locate(new_x, new_y, 2)
+	var/turf/topright = locate(new_x + LEADER_POD_DISPERSION, new_y + LEADER_POD_DISPERSION,2)
+	var/turf/bottomleft = locate(new_x - LEADER_POD_DISPERSION, new_y - LEADER_POD_DISPERSION,2)
+	var/list/block = block(bottomleft, topright) - locate()
```

Comment:
should the amount of space required (and amount of space checked) relate to how many occupied pods there are, instead of set numbers? That might make it better if you've got a large drop which I imagine is going to happen when this is merged.